### DISCUSSION TITLE
Difference of defineCollections and defineDocs

### DISCUSSION THREAD

**Discussion by Skreeauk** [fuma-nama/fumadocs#822] [Q&A] [answered]
I've been looking at the MDX v10 and I don't really understand why theres 2 of these functions

**Comment by fuma-nama** [comment, marked as answer]
> You can use defineDocs to define the required collections to work with Fumadocs. It offers the same API as defineCollections.


As the docs says, it's just a shortcut to define the required collections to work with Fumadocs itself

**Reply by Skreeauk** [reply]
I see, I was just really confused why theres like 2 functions that does the same thing. Thanks for clarifying.

On a sidenote, Does your MDX v10 come with any performance benefits? I saw the blog post and wanted to know if turbopack actually makes a huge difference compared to usual webpack

**Reply by fuma-nama** [reply]
They should have a similar speed because the previous Fumadocs MDX was performant and highly optimized.

We still have a few issues with Turbopack which affects some of our performance optimizations:
https://github.com/vercel/next.js/issues/69502 is fixed but not released yet.

if the fix get released, I'm sure there would be a noticeable improvement on dev mode
